<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>431</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="outputPlainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="optionLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="optionLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Option:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="optionsPlainTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>1 Enter, write it to file and print it
2 Enter array, write it to file and print it
3 Enter sport team information, write it to file and print it
4 Same as 3td, but write to binary file
5 Enter a sentence, write it to file and print it
6 Exit
</string>
    </property>
    <property name="overwriteMode">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="arraySizeLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Array size:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="arraySizeLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>300</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="arrayElementLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="arrayElementLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Array element</string>
    </property>
   </widget>
   <widget class="QLabel" name="sportTeamInfoLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sport team information:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ratingPosLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ratingPositionLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Rating position:</string>
    </property>
   </widget>
   <widget class="QLabel" name="winPercentageLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Win percentage:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="winPercentageLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>380</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="divisionLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>430</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Division (A-D):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="divisionLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="passedToChampionsLeagueLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="passedToChampionsLeagueLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Passed to champions</string>
    </property>
   </widget>
   <widget class="QLabel" name="passedToChampionsLeagueLabel_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>league (1 - yes, 0 - no):</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameAbbreviationLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>480</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name abbreviation</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameAbbreviationLabel_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>500</y>
      <width>170</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3 letters (AAA)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameAbbreviationLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>520</y>
      <width>170</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="playerNumberLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Player number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="playerNumberLineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="winPercentageLimitLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>340</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wrong percentage input</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
